M2/Macaulay2/packages/Macaulay2Doc/functions/installAssignmentMethod-doc.m2

```diff
@@ -56,7 +56,7 @@ document {
 	  x_c
      ///,
      "The source code explains how it works, behind the scenes.",
-     PRE ("    " | code {(symbol _,Symbol,Thing),((symbol _,symbol =),Symbol,Thing)}),
+     PRE ("    " | net code {(symbol _,Symbol,Thing),((symbol _,symbol =),Symbol,Thing)}),
```

Comment:
Why do you `net` this here?

Reply:
because at this stage, documentation is not ready to have non-Hypertext inside Hypertext, this is #2757.
I've managed to make it work since but I'd like to break this down into multiple PRs considering the amount of changes.

Reply:
I don't understand; isn't `code` returning hypertext now?

Reply:
precisely, it used to return `Net` which would just be copy/pasted into documentation. now it returns a `Hypertext` with potentially non-`Hypertext` in it, in the present case a `FilePosition`.